core/templates/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -26,47 +26,33 @@ import { ExplorationHtmlFormatterService } from
   'services/exploration-html-formatter.service';
 import { FractionObjectFactory } from 'domain/objects/FractionObjectFactory';
 import { HtmlEscaperService } from 'services/html-escaper.service';
-import { LoggerService } from 'services/contextual/logger.service';
+import {
+  ISubtitledHtmlBackendDict, SubtitledHtml, SubtitledHtmlObjectFactory
+} from 'domain/exploration/SubtitledHtmlObjectFactory';
+import { Interaction } from 'domain/exploration/InteractionObjectFactory';
 import { NumberWithUnitsObjectFactory } from
   'domain/objects/NumberWithUnitsObjectFactory';
-import { SubtitledHtml, SubtitledHtmlObjectFactory } from
-  'domain/exploration/SubtitledHtmlObjectFactory';
-import { UnitsObjectFactory } from 'domain/objects/UnitsObjectFactory.ts';
 
-export interface ExplanationBackendDict {
-  /* eslint-disable camelcase */
-  content_id: string;
-  /* eslint-enable camelcase */
-  html: string;
-}
-
-export interface SolutionBackendDict {
+export interface ISolutionBackendDict {
   /* eslint-disable camelcase */
   answer_is_exclusive: boolean;
-  correct_answer: string;
+  correct_answer: any;
```

Comment:
Why are we changing this from string to any?

Reply:
Because string was wrong, the answer can be of any type.

Reply:
Added a comment to explain.